core/drivers/crypto/hisilicon/sub.mk

```diff
@@ -1,3 +1,4 @@
 incdirs-y += include
 
 srcs-y += hisi_qm.c
+srcs-$(CFG_HISILICON_ACC_V3) += hpre_main.c
```

Comment:
Please preserve the newline character, that is, the file should end with a newline.